src/pages/test-report/cat-be/components/vehicle-checks/vehicle-checks.scss

```diff
@@ -1,4 +1,4 @@
-.test-report-vehicle-checks {
+vehicle-checks {
```

Comment:
Should we name this component `vehicle-checks-office`? The selector is called that, but maybe we could name the files that way as well